resources/webui.css

```
@@ -24,6 +47,12 @@ body {
   box-shadow: 0px 0px 3px 3px rgba(0, 0, 0, 0.2);
 }
 
+body.dark-theme .splash {
+  background-color: #191919;
+  border: 1px solid #444;
```

Let's use token variable first :)

```suggestion
  border: 1px solid var(--general-colorBorder, #444);
```